### PR TITLE
refactor(yaml): clean up unused values from YAML files

### PR DESCRIPTION
**Pull request by myii**
Don't merge this one blindly since there may be a couple of lines which you're intending to use.  I'll indicate the ones you may need to comment on.

### CODE REVIEW

File: nifi/osmap.yaml [hunk 1]
```diff
@@ -11,20 +11,12 @@
...
 Raspbian: {}
 
 # os_family: RedHat
-Fedora:
-  pkg:
-    name: nifi-fedora
-  service:
-    name: service-fedora
-CentOS:
-  config: '/opt/nifi/nifi.conf'
```
Comment (myii) [inline nifi/osmap.yaml:27]: Is `/opt/nifi/nifi.conf` actually used?  I don't see it accessed in the Travis runs.

**Comment by myii** [comment]
@codeboyQ2n5ha45 Just a general point about this: these YAML files are supposed to be used to supply the differences in configuration between the various platforms.  I've just cleaned them out completely, meaning that this formula isn't using these at all.  That suggests that some of the logic is taking place elsewhere unnecessarily.  I haven't got time to dig into this but it's another issue to keep in consideration.

**Comment by myii** [comment]
@codeboyQ2n5ha45 OK, like this block here for example:

https://github.com/codeboyQ2n5ha45/nifi-formula/blob/64ae7393605a129b8446642a3a885357626edb9d/nifi/package/javajdk.sls#L9-L33

Pretty much all of this Jinja is unnecessary, by supplying the values in the YAML files and then referencing those directly from the map instead, something like:

```sls
nifi-package-install-dependency-javajdk:
  pkg.installed:
    - name: {{ nifi.pkg_jdk }}
```

File: nifi/osmap.yaml (hunk 1 above)
Comment (codeboyQ2n5ha45) [reply nifi/osmap.yaml:27]: Ok. Yes. I was just testing here to see how config files work. This is not used.

**Comment by codeboyQ2n5ha45** [comment]
> 
> 
> @codeboyQ2n5ha45 OK, like this block here for example:
> 
> https://github.com/codeboyQ2n5ha45/nifi-formula/blob/64ae7393605a129b8446642a3a885357626edb9d/nifi/package/javajdk.sls#L9-L33
> 
> Pretty much all of this Jinja is unnecessary, by supplying the values in the YAML files and then referencing those directly from the map instead, something like:
> 
> ```yaml-salt
> nifi-package-install-dependency-javajdk:
>   pkg.installed:
>     - name: {{ nifi.pkg_jdk }}
> ```



> 
> 
> @codeboyQ2n5ha45 OK, like this block here for example:
> 
> https://github.com/codeboyQ2n5ha45/nifi-formula/blob/64ae7393605a129b8446642a3a885357626edb9d/nifi/package/javajdk.sls#L9-L33
> 
> Pretty much all of this Jinja is unnecessary, by supplying the values in the YAML files and then referencing those directly from the map instead, something like:
> 
> ```yaml-salt
> nifi-package-install-dependency-javajdk:
>   pkg.installed:
>     - name: {{ nifi.pkg_jdk }}
> ```

Ok. This is a bit over my head programming wise.

**Comment by codeboyQ2n5ha45** [comment]
I'm a bit confused, in what file exactly would I store the different values for the different package names for openjdk?

**Comment by myii** [comment]
@codeboyQ2n5ha45 Let me see if I can provide something.  In the meantime, once the `README` files have been merged into one, we can transfer this to the main org.  Please contact me via. IRC when you've done that.

**Comment by codeboyQ2n5ha45** [comment]
Ok. Great. So the README.md gets deleted? and I just copy the same contents into docs/README.rst?

**Comment by myii** [comment]
> Ok. Great. So the README.md gets deleted? and I just copy the same contents into docs/README.rst?

Yes, but you will need to adjust the format from Markdown to reStructuredText.  An simple way to do that is using an online converter such as:

* https://pandoc.org/try/?text=&from=gfm&to=rst

Instead of pushing direct to the `master` branch, which you won't be able to do once this repo is transferred, take this opportunity to [use a feature branch to set up a PR](https://gist.github.com/vlandham/3b2b79c40bc7353ae95a).  Ping me in the PR comment and I'll review the PR before it's merged.

**Comment by myii** [comment]
> Ok. This is a bit over my head programming wise. I might be able to spend a bit of time trying to figure out what you recommend this afternoon.

Solution proposed in #5.

**Pull request merged**